sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventDataBatch.java

```diff
@@ -30,8 +30,8 @@
  *
  * @see EventHubProducerClient#createBatch()
  * @see EventHubProducerAsyncClient#createBatch()
- * @see EventHubProducerClient See EventHubProducer for examples using the synchronous producer.
- * @see EventHubProducerAsyncClient See EventHubProducerAsyncClient for examples using the asynchronous producer.
+ * @see EventHubClientBuilder See EventHubClientBuilder for examples of building an asynchronous or synchronous
```

Comment:
Javadoc produces `See also EventHubClientBuilder` for `@see EventHubClientBuilder`. So, it would read `See also EventhubClientBuilder See EventHubClientBuilder for examples..`. 

`@see EventHubClientBuilder for examples of building an asynchronous or synchronous producer` should be sufficient.

Reply:
Fixed